crt_portal/cts_forms/templates/forms/complaint_view/index/grouping-select.html

```diff
@@ -1,4 +1,4 @@
 <select name="grouping" class="usa-select select-link usa-button--unstyled" aria-label="Grouping options">
   <option value="default">Default</option>
-  <option value="matching-descriptions">Matching descriptions</option>
+  <option onselect="gtag('event', 'click', {'event_name': 'group by matching description'});"  value="matching-descriptions">Matching descriptions</option>
```

Comment:
It'd be neat to capture section info with these.

We don't have a way to see what section the current user is from, but we could pass along the selected section filter?